docs/server/concepts/client_initialize_response.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: ClientInitializeResponse
+sidebar_label: ClientInitializeResponse
+---
+
+## Introduction
+
+Many customers come to us with strict performance requirements for their client applications, hoping they can still take advantage of the convience our client SDKs offer.  
+If you are looking to improve the performance of your Statsig integration on the client side, it may seem counter intuitive that a Server SDK can help.  Because every server SDK downloads
+the entire project definition, it can be used to generate values for your client side applications.  You can leverage this to integrate Statsig with server side rendering,
+or just to remove a network request to Statsig when your app loads (improving performance and solving the "flicker effect" at the same time).
+
+## Using a Server SDK
+
+This is one of the more complicated integrations to set up, and requires you to use Statsig on your backend as well.  If you're alreadying using one of our server SDKs, 
+its just a matter of 
```

Comment:
seems to trail off here
Perhaps link "server SDKs" to "https://docs.statsig.com/server/introduction"